src/grpc/grpc_server.cc

```diff
@@ -2435,6 +2435,99 @@ Server::Create(
   return nullptr;  // success
 }
 
+TRITONSERVER_Error*
+Server::Create(
+    std::shared_ptr<TRITONSERVER_Server>& server, UnorderedMapType& data,
```

Comment:
```suggestion
    std::shared_ptr<TRITONSERVER_Server>& server, UnorderedMapType& options,
```
`data` is not a descriptive name